packages/venia-ui/lib/RootComponents/Category/__tests__/__snapshots__/category.spec.js.snap

```diff
@@ -1,76 +1,40 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`it renders a loading indicator when appropriate 1`] = `
-<div>
-  <span>
-    <svg
-      fill="none"
-      height={64}
-      stroke="currentColor"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      strokeWidth="2"
-      viewBox="0 0 24 24"
-      width={64}
-      xmlns="http://www.w3.org/2000/svg"
-    >
-      <line
-        x1="12"
-        x2="12"
-        y1="2"
-        y2="6"
-      />
-      <line
-        x1="12"
-        x2="12"
-        y1="18"
-        y2="22"
-      />
-      <line
-        x1="4.93"
-        x2="7.76"
-        y1="4.93"
-        y2="7.76"
-      />
-      <line
-        x1="16.24"
-        x2="19.07"
-        y1="16.24"
-        y2="19.07"
-      />
-      <line
-        x1="2"
-        x2="6"
-        y1="12"
-        y2="12"
-      />
-      <line
-        x1="18"
-        x2="22"
-        y1="12"
-        y2="12"
-      />
-      <line
-        x1="4.93"
-        x2="7.76"
-        y1="19.07"
-        y2="16.24"
-      />
-      <line
-        x1="16.24"
-        x2="19.07"
-        y1="7.76"
-        y2="4.93"
-      />
-    </svg>
-  </span>
-  <span>
-    <mock-FormattedMessage
-      defaultMessage="Fetching Data..."
-      id="loadingIndicator.message"
-    />
-  </span>
-</div>
+Array [
```

Comment:
I'm not sure about this test - it says "renders a loading indicator when appropriate" but I'm not sure when appropriate _is_. The snap before seems to have the indicator whereas the snap after does not.

Reply:
Clarified by adding another test and updating the wording. The snapshot in question had changed because we're no longer rendering a full-page indicator when there's cached data.

Reply:
Ok that makes sense. Thanks for the update :)